nextcloudappstore/core/migrations/0022_auto_20200328_1055.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations, models
+import django.db.models.deletion
+import parler.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0021_nextcloudrelease_is_supported'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='app',
+            name='is_integration',
+            field=models.BooleanField(default=False, verbose_name='Integration (i.e. Outlook plugin)'),
+        ),
+        migrations.AlterField(
```

Comment:
@BernhardPosselt any ideas why all these migrations would appear for translations?

Reply:
Or maybe @blizzz ? :D